### PR TITLE
Fhac 197 FHIR Resource page for Admin GUI

### PR DESCRIPTION
**Pull request by jsmith2**
FHIR page allows user to update FHIR resource with new URL and to pull conformance information from that FHIR server.

### CODE REVIEW

File: Product/DBScripts/nhincdb/nhincdb.sql [hunk 1]
```diff
@@ -631,6 +635,6 @@ UPDATE domain SET postmasterAddressId = 2 WHERE id = 1;
 
 USE eventdb;
 LOCK TABLES event WRITE;
-ALTER TABLE event MODIFY COLUMN eventTime timestamp null;
+ALTER TABLE event MODIFY COLUMN eventTime timestamp not null;
```
Comment (christophermay07) [inline Product/DBScripts/nhincdb/nhincdb.sql]: Can these eventdb statements be removed altogether?  The timestamp type is not null by default.

File: Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml [hunk 2]
```diff
@@ -215,6 +215,15 @@
             <groupId>org.mockito</groupId>
             <artifactId>mockito-all</artifactId>
         </dependency>
+        
+        <!-- FHIR -->
+        
+        <dependency>
+            <groupId>me.fhir</groupId>
+            <artifactId>fhir-0.81</artifactId>
+            <version>2382-SNAPSHOT</version>
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml:224]: Should we manage the version through Product/pom.xml?

Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml:224]: No, not used anywhere else.  And no plans yet to use it anywhere else.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/client/fhir/ConformanceClient.java [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                throw new EFhirClientException("Error parsing response message", e);
+            } finally {
+                StreamUtils.closeStreamSilently(instream);
+            }
+        }
+        if (resource instanceof OperationOutcome) {
+            if (((OperationOutcome) resource).getIssue().size() > 0) {
+                throw new EFhirClientException((OperationOutcome) resource);
+            } else {
+                System.out.println(((OperationOutcome) resource).getText().getDiv().allText());//TODO change to formal logging
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/client/fhir/ConformanceClient.java:133]: Multiple unhandled TODO's in this PR

Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/client/fhir/ConformanceClient.java:133]: Code was used from the reference client.  Ignore for now.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/client/fhir/ConformanceClient.java [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        AtomEntry<T> atomEntry = buildAtomEntry(response, resource);
+        return new ResourceRequest<T>(atomEntry, response.getStatusLine().getStatusCode());
+    }
+
+    protected static void configureFhirRequest(String resourceFormat, HttpRequest request) {
+        request.addHeader("User-Agent", "Java FHIR Client for FHIR");
+
+        LOG.info("Resource format: " + resourceFormat + ", Feed Format: " + FeedFormat.FEED_XML.getHeader());
+        request.addHeader("Accept", resourceFormat);
+        request.addHeader("Accept", FeedFormat.FEED_XML.getHeader());
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/client/fhir/ConformanceClient.java]: The "Accept" header is set twice, is this intentional?

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without further comment.

**Comment by nareshcgi** [comment]
Reviewed the PR and looks good.

**Pull request merged**